README.md

```
@@ -77,6 +82,35 @@ Show output from within test steps. "Output" is `print`, `echo`, `var_dump`, etc
 - `on-fail` - only show the output if there are test fails
 - `in-summary` - only show in the summary if there are test fails
 
+## Migrating to Behat core
+
+Upgrade Behat, remove this extension from your `behat.yml`, and enable the option on the built-in `progress` formatter instead:
+
+>behat.yml
+```yaml
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)